frontend/src/components/Utils/utils.test.js

```diff
@@ -0,0 +1,37 @@
+import * as utils from './utils';
+
+test('new york date timestamp format', () => {
+  // Month parameter is zero indexed so it's actually the 10th month.
```

Comment:
This comment is good, but it may be better to only include it once in the first test or include it in every test. Having it in 2 tests but not the other 2 is a little confusing.